shared/actions/settings.tsx

```diff
@@ -614,7 +614,10 @@ const loadHasRandomPW = async (state: TypedState) => {
 }
 
 // Mark that we are not randomPW anymore if we got a password change.
-const passwordChanged = () => SettingsGen.createLoadedHasRandomPw({randomPW: false})
+const passwordChanged = async (_: TypedState, action: EngineGen.Keybase1NotifyUsersPasswordChangedPayload) => {
+    const randomPW = action.payload.params.state === RPCTypes.PassphraseState.random
```

Comment:
should I put a try {} catch {} around here like in loadHasRandomPw? Or is that only for RPC calls?

Reply:
that's only for RPC calls